src/main/webapp/mentors.js

```diff
@@ -6,33 +9,27 @@ function getMentors() { // eslint-disable-line no-unused-vars
   console.log('entering get mentors function/n');
   let url = new URL(
       '/mentors', window.location.protocol + '//' + window.location.hostname);
-
-  fetch(url).then(errorHandling).then((response) => response.json())
-      .then((mentors) => {
+  const fetchRequest = standardizeFetchErrors(
+        fetch(url),
+        'Failed to communicate with the server, please try again later.',
+        'Encountered a server error, please try again later.');
+  
+  fetchRequest.then((mentors) => {
+    console.log('got mentor data');
     const mentorsContainer = document.getElementById('mentors-container');
     mentorsContainer.innerHTML = '';
     for (const mentor of mentors) {
       console.log(mentor);
       mentorsContainer.appendChild(createMentorElement(mentor));
     }
   })
-  .catch((error) => {
-    console.log(error);
+  .catch((errorResponse) => {
+      alert(errorResponse.userMessage);
+      console.error(
+          `Error ${errorResponse.statusCode}: ${errorResponse.error}`);
   });
```

Comment:
Nice! Just wanted to let you know that some of this behavior is going to change with #72